src/ex4/server.js

```diff
@@ -0,0 +1,25 @@
+import express from "express";
+import favicon from "serve-favicon";
+import { logger } from "./server/middlewares/logger.js";
+import { todoRouter } from "./server/routes/todoRouter.js";
+import { fileURLToPath } from "url";
+import { dirname, join } from "path";
+import cors from "cors";
+
+const port = 8000;
+
+const app = express();
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+app.use(favicon(join(__dirname, "dist", "images", "favicon.ico")));
+
+app.use(express.json());
+app.use(cors());
+app.use(logger);
+
+app.use("/dist", express.static("dist"));
```

Comment:
you can change the first param to be `/` then the client will be accessible via `localhost:8000` instead of `localhost:8000/dist`